ganttproject/src/biz/ganttproject/impex/google/GoogleAuth.java

```diff
@@ -30,76 +31,54 @@
   private static final String APPLICATION_NAME =
           "Google Export for GanttProject";
 
-  // Directory to store user credentials for this application
-  private static final java.io.File DATA_STORE_DIR = new java.io.File(
-          "data/resources/credentials");
-
-  // Global instance of the {@link FileDataStoreFactory}
-  private static FileDataStoreFactory DATA_STORE_FACTORY;
-
   // Global instance of the JSON factory
-  private static final JsonFactory JSON_FACTORY =
+  private final JsonFactory JSON_FACTORY =
           JacksonFactory.getDefaultInstance();
 
   // Global instance of the HTTP transport
-  private static HttpTransport HTTP_TRANSPORT;
+  private HttpTransport HTTP_TRANSPORT = new HttpTransport() {
```

Comment:
Please remove it and construct HttpTransport exactly where you need it, without holding in a member/static/global or whatever variable